nifi-nar-bundles/nifi-standard-services/nifi-hbase-client-service-api/src/main/java/org/apache/nifi/hbase/HBaseClientService.java

```diff
@@ -126,6 +138,25 @@
 
     void delete(String tableName, List<byte[]> rowIds) throws IOException;
 
+    /**
+     * Deletes a list of cells from HBase. This is intended to be used with granual delete operations.
```

Comment:
granular?

Reply:
I think that's a reasonable word choice because deleting whole row is not necessarily possible all at once with visibility labels enabled. Each cell that has a label must have its visibility label sent as part of the delete request. So you have to build a nuanced delete request.

Reply:
Sorry, I was not arguing the wording, just mentioning a possible typo :)